src/LostAndFound/Game.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LostAndFound
+{
+    public static class Game
```

Comment:
Later we will probably need to rename this, but I can't think of a better name right now.